packages/web/src/components/track/DownloadRow.tsx

```diff
@@ -105,10 +114,14 @@ export const DownloadRow = ({
               whiteSpace: 'nowrap'
             }}
           >
-            {getDownloadFilename({
-              filename: filename ?? track?.orig_filename,
-              isOriginal: true
-            })}
+            {filename ??
+              (track &&
+                user &&
+                getFilename({
```

Comment:
nit: what happens when track or user don't exist? could use ternary to be explicit

Reply:
yeah fair enough!